README.md

```diff
@@ -13,7 +13,11 @@ Dotenv is a zero-dependency module that loads environment variables from a `.env
 ## Install
 
 ```bash
-npm install dotenv --save
+# with npm
+npm i dotenv
```

Comment:
does `npm` save by default now? if not, should keep `--save` flag. I also have a strong preference for `install` over `i` for documentation for anyone not familiar with shorthand

Reply:
Yep, npm saves by default now. And yeah I agree with @maxbeatty that the explicit long-from command names are better for documentation

Reply:
Yes npm saves to `package.json` by default now.